src/library/styles/styleReportOverview.js

```diff
@@ -2,10 +2,10 @@ import {StyleSheet} from 'react-native';
 import {colors} from '../../../src/library/styles/vars';
 const style = StyleSheet.create({
   reportDetails: {
-    marginBottom:5,
-    backgroundColor:  '#FFFFFF',
+    marginBottom: 5,
+    backgroundColor: '#FFFFFF',
```

Comment:
I notice that you use constants for colors, here you could also use a constant for the color black.